packages/sdks/web-component/src/lib/descope-wc/sdkScripts/fingerprintDescope.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  load,
+  defaultEndpoint,
+  defaultScriptUrlPattern,
+} from '@fingerprintjs/fingerprintjs-pro';
+
+export const loadFingerprint = async (
+  initArgs: {
+    customDomain?: string;
+    publicApiKey: string;
+    cloudflareEndpointPath: string;
+    cloudflareScriptPath: string;
+  },
+  inputs: { baseUrl?: string },
+  onTokenReady: (token: string) => void,
+) => {
+  try {
+    const {
+      customDomain,
+      publicApiKey,
+      cloudflareEndpointPath,
+      cloudflareScriptPath,
+    } = initArgs;
+    const { baseUrl } = inputs;
+    let fpUrl: string;
+    if (customDomain) {
+      fpUrl = `https://${customDomain}`;
+    } else if (baseUrl) {
+      fpUrl = baseUrl;
+    } else {
+      fpUrl = 'https://api.descope.com';
+    }
+
+    const endpointUrl = new URL(fpUrl);
+    endpointUrl.pathname = cloudflareEndpointPath;
```

Comment:
how will `cloudflareScriptPath` will be configured for FP-Descope?

Reply:
it's an env var in the orchestration service. 
see https://github.com/descope/orchestrationservice/pull/2536/files#diff-8343793a64c62887287e0891045c62285d84954ba5ab10bf6579a99ab91e0100R224

Reply:
is it possible that we will have `cloudflareEndpointPath` with 
a) developer provided `baseUrl`
b) `https://api.descope.com`

?